docs/design.md

```diff
@@ -209,27 +209,27 @@ on a node is at a timestamp < next HLC time.
 
 Transactions are executed in two phases:
 
-1. Start the transaction by writing a new entry to the system
-   transaction table (keys prefixed by *\0tx*) with state “PENDING”. In
+1. Start the transaction by selecting a range which is likely to be
```

Comment:
s/is likely to be heavily involved in the transaction/involved in the first write of the transaction/